Atomic/backendutils.py

```diff
@@ -156,6 +156,13 @@ def get_containers(self):
             con_objs += be.get_containers()
         return con_objs
 
+    def get_container_ids_from_image(self,image):
```

Comment:
I did that because we can get both docker images and ostree images from the backendutils